[2023-05-04T22:04:50.851+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_pipeline.feature_selection.feature_selecting manual__2023-05-04T22:03:13.940561+00:00 [queued]>
[2023-05-04T22:04:50.864+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_pipeline.feature_selection.feature_selecting manual__2023-05-04T22:03:13.940561+00:00 [queued]>
[2023-05-04T22:04:50.865+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-05-04T22:04:50.897+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-05-04T22:04:50.898+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-05-04T22:04:50.936+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): feature_selection.feature_selecting> on 2023-05-04 22:03:13.940561+00:00
[2023-05-04T22:04:50.943+0000] {standard_task_runner.py:55} INFO - Started process 22466 to run task
[2023-05-04T22:04:50.946+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'ml_pipeline', 'feature_selection.feature_selecting', 'manual__2023-05-04T22:03:13.940561+00:00', '--job-id', '676', '--raw', '--subdir', 'DAGS_FOLDER/ml_pipeline.py', '--cfg-path', '/tmp/tmptopft6ly']
[2023-05-04T22:04:50.949+0000] {standard_task_runner.py:83} INFO - Job 676: Subtask feature_selection.feature_selecting
[2023-05-04T22:04:51.029+0000] {task_command.py:389} INFO - Running <TaskInstance: ml_pipeline.feature_selection.feature_selecting manual__2023-05-04T22:03:13.940561+00:00 [running]> on host ab75f0ddb6b9
[2023-05-04T22:04:51.121+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=nomadicsenseis@gmail.com
AIRFLOW_CTX_DAG_OWNER=Diego Rivera
AIRFLOW_CTX_DAG_ID=ml_pipeline
AIRFLOW_CTX_TASK_ID=feature_selection.feature_selecting
AIRFLOW_CTX_EXECUTION_DATE=2023-05-04T22:03:13.940561+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-04T22:03:13.940561+00:00
[2023-05-04T22:04:54.319+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/utils/validation.py:993 DataConversionWarning: A column-vector y was passed when a 1d array was expected. Please change the shape of y to (n_samples, ), for example using ravel().
[2023-05-04T22:05:13.483+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:05:32.125+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:05:51.635+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:06:13.049+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:06:36.938+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:07:00.565+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:07:27.629+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:07:51.744+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:08:12.813+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:08:33.733+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:08:54.003+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:09:14.149+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:09:33.648+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:09:49.410+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:10:04.721+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:10:19.738+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:10:35.137+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:10:50.272+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:11:05.667+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:11:21.421+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:11:36.763+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:11:52.400+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:12:08.024+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:12:23.471+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:12:38.363+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:12:53.703+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:13:09.050+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:13:24.881+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:13:40.273+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:13:56.265+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:14:13.704+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:14:29.515+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:14:45.190+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:15:00.659+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:15:15.873+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:15:31.130+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:15:47.277+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:16:03.323+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:16:19.274+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:16:35.112+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:16:50.902+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:17:06.429+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:17:22.213+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:17:37.565+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:17:52.767+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:18:08.046+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:18:24.231+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:18:40.160+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:18:56.221+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:19:12.228+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:19:32.410+0000] {logging_mixin.py:137} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/linear_model/_sag.py:354 ConvergenceWarning: The max_iter was reached which means the coef_ did not converge
[2023-05-04T22:19:35.026+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-05-04T22:19:35.039+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=ml_pipeline, task_id=feature_selection.feature_selecting, execution_date=20230504T220313, start_date=20230504T220450, end_date=20230504T221935
[2023-05-04T22:19:35.088+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-05-04T22:19:35.130+0000] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
